tasks/test/mocha-chrome.js

```diff
@@ -16,7 +16,7 @@ module.exports = function(grunt) {
     // Launch Chrome in headless mode
     const browser = await puppeteer.launch({
       headless: true,
-      args: ['--no-sandbox']
+      args: ['--no-sandbox', '--disable-setuid-sandbox']
```

Comment:
In order for the test suite to work this change has to be merged.

Reply:
It's probably fine to add but I would like to test this on a Linux machine before commiting it.

Reply:
for anyone coming here later, this change was added to `main` in https://github.com/processing/p5.js/pull/4959/commits/e9f04db7880a797afae3d6e13862f0ddd2fef592